.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        suite: [test, lint]
```

@PiotrKozlowski 
this is what you mean?
```suggestion
        suite: [test, lint, build]
```

---

yes